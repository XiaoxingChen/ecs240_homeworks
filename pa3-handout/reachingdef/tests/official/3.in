c Straightline code (large)
p 200 100 99 1 100
b 1
e 1 2
b 2
e 2 3
b 3
e 3 4
b 4 22
e 4 5
b 5
e 5 6
b 6 78 31 55 89 60 27 62 15 100 83 44 21 57 71 72 22 89 41 4 29 2 88 3 100 97 65 74 17 94 97 93 22 49 39 88 27 54 61 66 64 33 54 2 6 47 23 93 19 48 93 12 69 95 40 50 69 7 90 50 44 18 37 21 71 73 17 79 34 15 32 8 66 16 77 18 74 89 94 51 42 21 38 20 77
e 6 7
b 7
e 7 8
b 8
e 8 9
b 9
e 9 10
b 10 29 84 1 22 76 98 17 18 92
e 10 11
b 11
e 11 12
b 12
e 12 13
b 13
e 13 14
b 14 99 41 95 59 13 7 39 52 65 78 26 45 79 63 92 66 19 24 68 54 32 85 6 60 6 63
e 14 15
b 15
e 15 16
b 16
e 16 17
b 17 96 61 88 29 15 64 95 32 80 7 97 6 88 62 100 47 57 2 66 17 84 93 17 23 60 23 62 90 89 34 13 16 94 82 23 72 22 45 24 54 28 24 70 67 87 97 92 47 59 23 95 18 17 2 57 67 66 8 8 80 22 28 20 87 58 73 52 48 4 96 20 24 16 10 31 65 82 5 63 15 60 95 82 81 46 45 65 77 90 84 82 11 42
e 17 18
b 18 25 4 9 31 15 68 70 17 78 90 63 70 50 90 66 18 86 19 32 83 77 60 1 53 41 5 64 22 1 42 95 23 35 35 18 96 84 74 34 67 54 57 49 10 20 72 94 83 46 44 78 4 52 51 67 60 24 40 56 78 23 34 5 50 76 76 25 25 100 63 22 41 27 93 14 25 86
e 18 19
b 19 78 89 6 14 52 55 69 64 21 65 65 17 12 41 89 23 11 28 75 21 71 23 82 27 65 54 84 19 79 34 66 78 73 75 64 47 48 81 3 11 13 67 13 15 2 93 93 87 64 65 73 58 86 70 34 10 92 9 67 19 68 91 60 48 22 97 96 78 77 52 54 25 34 77 67 47 96 100
e 19 20
b 20 51 38 95 14 30 99 82 14 29 8 96 46 20 25 45 93 49 99 46 4 100 4 91 33 95 17 3 99 5 27 46 83 91 52 14 83 97 93 91 29 87 54 18 70 5
e 20 21
b 21 63 13 52 99 90 30 12 81 93 22 44 65 32 24 2 69 71 89 15 38 58 42 50 68 19 3 4 61 13 26 71 86 60 99 30 37 13 72 96 66 11 86 1 81 58 37 70 27 91 36 39 2 52 29 83 36 29 89 15 58 35 87 1 26 47 10 8 32 35 91 55 30 92 97 44 48 100 99 57 60 53 89 64 34 79
e 21 22
b 22
e 22 23
b 23 79 1 82 28 21 45 21 33 8 97 74 14 23 10 73 27 90 60 95 72 42 23 80 61 10 51 8 45 72 2 61 77 91 41 90 71 51 18 51 67 43 27 61 38 74 46 56 51 80 12 27 55 70 33 27 78 50 39 27 20 60 57 99 64 63 53 9 12 74 91 41 92 16 58 19 42 80 12 41 60 64 86 58 78 48 64 44 8 42 54 60 82 44 17 99 23 18
e 23 24
b 24
e 24 25
b 25
e 25 26
b 26
e 26 27
b 27
e 27 28
b 28 74 75 67 70 31 5 49 63 5 81 27 86 32 11 15 13 48 88
e 28 29
b 29 94 62 53 94 81 68 83 94 44 85 29 87 21 93 1 39 89 90 100 47 21 85 100 9 62 69 18 26 53 24 52 92 1 42 89 68 26 27 2 80 23 33 63 22 73 92 97 9 40 30 37 25 16 90 19 24 22 53 78 57 3 83 50 58 33 19 82 68 74 88 76 82 40 98 35 6 52 48 16 37 45 63
e 29 30
b 30
e 30 31
b 31 60 63 54 67 3 79 13 57 36 88 7 37 42 28 54 22 58 47 76 95 52 72 15 50 3 46 25 24 66 82 53 9 81 95 36 93 72 59 2 78 14
e 31 32
b 32 79 23 50 86 61 38 28 32 56 57 87 48 85 89 72 37 43 48 47 48 7 25 68 27 67 67 86 48 16 83 31 11 95 86 71 41 64 5 70 42 77 35 27 34 14 38 92 74 78 60 7
e 32 33
b 33 2 67 55 66 57 58 39 29 6 87 42 10 2 64 33 29 48 30 66 42 11 9 64 47 72 61 40 63 74 17 11 35 47 87 6 77 5 27 44 75 66 10 29 18 49 1 79 24 56 85 18 36 3 4 40 46 29 84 70
e 33 34
b 34 68 70 25 91 34 51 90 48 32 65 95 3 66 43 46 33 9 73 20 54 83 16 21 54 87 74 93 96 12 90 24 63 70 77 15 70 18 15 61 72 26 12 35 76 55 93 32 52 79 83 56 89 3 2 26 87 65 93 78 10 17 38 9 33 60
e 34 35
b 35 60 78 52 94 84 16 1 60 99 49 44 84 59 71 63 66 28 72 37 9 58 82 21 49 59 74 84 96 61 90 73 40 26 3 47 52 29 65 91 82 50 11 97 44 63 84 16 27 19 6 83 73 61 73 13 94 46 6 76 11 11 70 21 59 61 66 53 11 24 3 74 43 10 71 62
e 35 36
b 36 18 97 99 4 14 56 53 15 58 88 57 53 100 98 62 77 6 92 8 4 39 54 43 26 47 17 74 24 30 31 61 17 10 88 95 88 52 85 51 37 15 94 2 97 98 34 76 17 14 5 75 38 81 85 91 29 23 71 4 51 93 25 63 100 77 97 88 18 87 66 79 69 70 9 84 91 81 20 82 55 38 80 85
e 36 37
b 37 5 39 57 33 66 19 82 25 13 25 94 19 62 71 58 46 65 11 56 79 45 68 62 85 69 32 29 66 68 73 14 72 89 56 55 13 45 64 76 17 76 77 6 54 66 46 35 86 54 89 78 80 44 56 53 7 55 100 68 38 69 37 96 86 6 20 13 83 43 58 29 75 83 59 17 22 21 14 3 36
e 37 38
b 38
e 38 39
b 39 18 17 23 16 27 35 51 76 86 64 21 17 28 73 87 28 20 14 86 27 78 29 42 95 61 60 59 90 10 94 17 61 80 82 41 3 38 64 85 58 52 77 28 60 74 100 30 61 81 2 23 23 37 73 66 9 52 7 38 42 9 65
e 39 40
b 40 66 98 50 71 81 82 58 30 30 31 67 1 78 97 82 58 28 38 54 97 16 73 90 78 98 23 6 33 36 54 73 39 26 57 10 64 55 19 66 39 39 49 8 7 12 88 22 18 82 34 1 2 55 48 31 8 90 35 14 74 100 57 26 94 40 27 77 4 76 75 4 25 21 80 57 99 64 26 9 11 99 67 92 20 51
e 40 41
b 41 21 9 50 50 63 65 61 21 77 24 18 20 42 34 12 53 99 57 21 76 73 4 78 45 3 40 34 5 49 47 88 10 80 11 39 8 34 33 16 54 18 25 32 4 92 57 89 46 75 72 21 28 91 90 3 76 98 14 55 32 100 68 46 35 4 17 90 78 56 2 90
e 41 42
b 42
e 42 43
b 43 100 22 56 15 86 18 56 41 74 79 75 95 1 18 30 27 67 12 84 2 14 5 32 40 17 34 72 98 86 35 18 59 33 4 90 92 22 47 27 62 11 34 100 27 68 89 8 74 3 58 53 83 33 2 22 64 52
e 43 44
b 44 38 47 16 48 55 27 48 69 57 88 42 4 47 33 20 68 97 68 11 60 14 64 8 63 75 30 90 32 17
e 44 45
b 45
e 45 46
b 46
e 46 47
b 47 83 61 23 30 38 64 53 2 67 18 91 40 33 10 83 5 72 32 12 26 7 95 32 51 83 36 28 15 100 95
e 47 48
b 48
e 48 49
b 49 94 17 10 74 89 83 6 80 57 96 48 33 41 76 35 8 57 54 28 36 14 90 3 15 68 31 71 92 27 49 21 26 58 5 11 94 66 52 96 75 47 2 78 5 36 2 30 56 78 72 90 21 19 45 51 43 61 34 17 46 20 56 26 33 1 91 7 44 95 21 84 55 46 23 75 97 50 21 82
e 49 50
b 50 88 50 71 92 23 4 80 27 21 92 99 20 53 14 78 12 78 69 54 16 92 23 2 25 48 64 72 67 63 43 65 4 27 55 63 98 63 71 83 41 38 83 6 4 8 76 12 64 42 22 33 26 41 26 68 78 55 12 24 94 69 95 58 68 87 40 60 73 33 11
e 50 51
b 51 94 40 30 56 61 32 81 34 91 91 86 5 2 55 33 20 21 22 32 93 22 49 57 75 52 49 19 24 92 72 5 92 90 92 48 48 55 78 5 40 44 87 40 8 36 36 88 12
e 51 52
b 52
e 52 53
b 53
e 53 54
b 54 73 15 71 63 37 55 57 9 78 97 39 18 13 21 53 56 21 13 85 51 34 21 91 70
e 54 55
b 55 23 46 18 3 6 73 94 93 6 94 80 38 80 25 66 31 90 82 11 28 24 88 17 78 93 99 12 24 54 53 38 74 50 25 25 70 33 46 86 33 27 2 13 29 62 73 73 14 51 12 40 27 17 94 66 96
e 55 56
b 56
e 56 57
b 57 14 75 21 32 73 59 96 38 85 93 83 87 79 24 73 81 42 69 87 5 2 13 78 17 75 93 33 13 31 74 62 91 70 74 21 24 56 32 41 7 58 64 14 61 85 93 85 55 27 86 28 24 22 37 20 92 43 21 14 1 69 85 46 88 87 40 61 96 49 47 8 32 18 99 84 62 97 91 98 94 82 72 17 6 35 58 22 11 91
e 57 58
b 58
e 58 59
b 59 59 75 35
e 59 60
b 60
e 60 61
b 61 31 27 31 64 57 66 36 20 17 12 65 46 55 97 45 97 21 12 5 25 67 29 22 40 59 7 14 16 54 21 71 89 74 32 48 52 7 71 70 39 16 76 79 53 89 11 44 65 83 76 23 82 46 70 18 63 90 98 49 78 58 35 75 76 18 100 91 40 13 12 7 96 33 74 91 8 92 14 43 77 45 16 19
e 61 62
b 62
e 62 63
b 63
e 63 64
b 64
e 64 65
b 65
e 65 66
b 66 27 61 23 38 79 11 92 78 5 19 2 54 38 75 17 21 96 74 34 65 54 10 10 29 26 78 97 38 22 11 19 93 83 10 34 22 89 86 87 47 13 7 76 34 77 66 17
e 66 67
b 67
e 67 68
b 68 86 100 1 47 18 9 3 47 26 42 99 6 14 91 95 93 9 75 6 49 16 30 78 37 42 56 13 92 47 47 23 49 7 2 50 45 86 10 42
e 68 69
b 69 85 77 82 80 42 62 3 24 42 94 52 95 23 30 77 60 86 32 64 70 19 7 70 10 50 72 37 34 100 20 42 58 45 39 89 72 19 88 67 65 95 39 39 49 83 26 52 19 77 54 59 77 3 35 36 99 58 61 83 59 14 34 48 34 97 25 13 68 23 28 32 46 11 77 80 66 78 96 12 13 34 43 37 83 59 34 22 9 14 39 91 12 1 28 78 29 46 86 72 72
e 69 70
b 70
e 70 71
b 71
e 71 72
b 72
e 72 73
b 73 22 83 96 57 38 67 23 2 5 43 77 60 31 75 4 73 39 25 89 80 42 10 72 79 22 23 13 78 68 42 64 33 2 32 42 81 93 4 35 92 50 7 89 26 88 68 61 91 8 57 94 78 86 29 28 95 43 46 69 35 81 29 27 91 53 61 94 96 75 44 21 12 12 53 44 14 42 92 74 91 1 25 80 41 23 43 96 31 14 48 48 8 15 77 74 34 27 96 49 51
e 73 74
b 74
e 74 75
b 75 29 7 61 1 13 6 96 26 88 83 43 24 93 75 51 66 51 30 100 4 95 95 10 93 31 59 60 22 33 99 64 53 33 20 39 94 81 82 100 80 44 78 2 68 55 23 97 48 92 14 81 19 59 24 48 64 40 62 67 84 57 48 47 6 28
e 75 76
b 76
e 76 77
b 77
e 77 78
b 78
e 78 79
b 79
e 79 80
b 80
e 80 81
b 81
e 81 82
b 82 6 46 25 16 50 82 21 32 33 43 91 21 91 29 43 74 28 73 24 51 62 77 98 42 78 19 70 91 84 70 86 39 94 9 49 3 41 47 18 41 67 57 60 95 33 48 78 29 83 34 78 96 97 4 80 92 18 60 62 19
e 82 83
b 83 100 17 90 57 25 68 71 16 71 23 84 29 42 39 40 8 30 27 28 78 79 17 17 19 50 20 75 52 8 98 37 57 100 13 84 95 75 61 42 96 46 75 24 87 66 27 57 47 67 47 74 65 29 45 70 8 13 28 29 80 59 23 17 75 95 21 27 51 90 19 80 44 44 78 45 13 70
e 83 84
b 84 9 83 98 76 50 34 94 52 17 44 18 10 18 17 29 13 71 79 2 65 99 76 14 72 65 6 61 86 73 56 91 53 61 37 85 2 34 70 21 76 83 41 72 13 47 35 47 7 56 95 6 61 68 25 7 6 76 71 37 75 99 71 43 46 9 77 93 81 65 50 28 14 62 29 25 29 32 59 44 98 8 97 65 19 72 41 74 52 30 85 50 21 97 10 35 50 89 38 90 91 6
e 84 85
b 85
e 85 86
b 86 98 85 12 69 24 61 43 19 18 68 79 72 42 6 13 73 29 52 43 73 49 56 18 66 89 81 96 75 59 2 55 3 73 84 9 14 18 38 76 95 6 9 44 37 29 46 82 87 76 69 52
e 86 87
b 87 42 49 43 34 83 1 32 50 99 94 55 75 94 93 68 98 94 89 83 59 27 14 48 50 65 58 15 67 58 34 9 29 74 83 89 4 85 89 37 39 72 59 72 57 33 70 16 8 20 68 76 19 28 13 69 35 66 15 13 48 16 98 14 99 86 1 49 83 69 53 82 58 40 59 38 74 10 47 74 93 73 19 7 12 55 22 55 91 72 45
e 87 88
b 88 30 68 81 27 5 94 39 55 61 79 10 30 96 97 8 24 98 47 10 5 9 46 41 55 18 13 10 46 76 51 88 82 68 53 56 98 78 79 3 91 11 15 89 81 87 66 11 21 10 7 60 20 94 52 15 71 83 45 75 68 76 87 90 38 62 90 42 93 29 42 90 84 13 14 60 76 27 72 77 36 2 75 81 12 68 55 86 88 19 2 72 29 47 8 16 25 91 21 81
e 88 89
b 89
e 89 90
b 90
e 90 91
b 91
e 91 92
b 92 38 45 84 3 100 1 95 3 10 96 29 71 75 92 90 79 98 19 17 8 27 68 89 18
e 92 93
b 93
e 93 94
b 94
e 94 95
b 95 71 14 94 94 23 32 24 39 5 97 35 98 75 87 43 49 82 44 37
e 95 96
b 96 58 95 8 73 100 44 75 95 4 70 64 7 88 91 11 81 98 97 14 51
e 96 97
b 97 43 73 24 86 57 7 86 16 52 47 45 6 62 53 71 27 7 71 69 73 44 21 13 69 10 64 39 5 29 91 55 21 99 14 5 69 38 7 61 89 16 17 49 94 39 30 40 16 46 60 43 77 96 4 42 13 38 55 2 49 10 31 89 73 83 36 47 4 61 52 2 15 97 5 56 76 81 72 11 9 9 87 22 95
e 97 98
b 98
e 98 99
b 99
e 99 100
b 100