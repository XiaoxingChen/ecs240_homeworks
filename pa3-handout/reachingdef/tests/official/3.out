rdout 1
rdout 2
rdout 3
rdout 4 4
rdout 5 4
rdout 6 4 6
rdout 7 4 6
rdout 8 4 6
rdout 9 4 6
rdout 10 4 6 10
rdout 11 4 6 10
rdout 12 4 6 10
rdout 13 4 6 10
rdout 14 4 6 10 14
rdout 15 4 6 10 14
rdout 16 4 6 10 14
rdout 17 4 6 10 14 17
rdout 18 4 6 10 14 17 18
rdout 19 4 10 14 17 18 19
rdout 20 4 10 14 17 18 19 20
rdout 21 4 10 14 17 18 19 20 21
rdout 22 4 10 14 17 18 19 20 21
rdout 23 4 10 14 17 18 19 20 21 23
rdout 24 4 10 14 17 18 19 20 21 23
rdout 25 4 10 14 17 18 19 20 21 23
rdout 26 4 10 14 17 18 19 20 21 23
rdout 27 4 10 14 17 18 19 20 21 23
rdout 28 4 10 14 17 18 19 20 21 23 28
rdout 29 4 10 14 17 18 19 20 21 23 28 29
rdout 30 4 10 14 17 18 19 20 21 23 28 29
rdout 31 4 10 14 17 18 19 20 21 23 28 29 31
rdout 32 4 10 14 17 18 19 20 21 28 29 31 32
rdout 33 4 10 14 17 18 19 20 21 28 29 31 32 33
rdout 34 4 10 14 17 18 19 20 21 28 29 31 32 33 34
rdout 35 4 10 14 17 18 19 20 21 28 29 32 33 34 35
rdout 36 4 10 14 17 18 19 20 21 28 29 32 33 34 35 36
rdout 37 4 10 14 17 18 19 20 21 28 29 32 33 34 35 36 37
rdout 38 4 10 14 17 18 19 20 21 28 29 32 33 34 35 36 37
rdout 39 4 10 14 17 18 19 20 21 28 29 32 33 34 35 37 39
rdout 40 4 10 14 17 18 19 20 21 28 29 32 33 34 35 37 39 40
rdout 41 4 10 14 17 18 19 20 21 28 29 32 33 34 35 37 39 40 41
rdout 42 4 10 14 17 18 19 20 21 28 29 32 33 34 35 37 39 40 41
rdout 43 4 10 14 17 18 19 20 21 28 29 32 33 34 35 37 39 40 41 43
rdout 44 4 10 14 17 18 19 20 21 28 29 32 33 34 35 37 39 40 41 43 44
rdout 45 4 10 14 17 18 19 20 21 28 29 32 33 34 35 37 39 40 41 43 44
rdout 46 4 10 14 17 18 19 20 21 28 29 32 33 34 35 37 39 40 41 43 44
rdout 47 4 10 14 17 18 19 20 21 28 29 32 33 34 35 37 39 40 41 43 44 47
rdout 48 4 10 14 17 18 19 20 21 28 29 32 33 34 35 37 39 40 41 43 44 47
rdout 49 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 49
rdout 50 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 49 50
rdout 51 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51
rdout 52 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51
rdout 53 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51
rdout 54 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54
rdout 55 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55
rdout 56 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55
rdout 57 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57
rdout 58 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57
rdout 59 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59
rdout 60 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59
rdout 61 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61
rdout 62 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61
rdout 63 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61
rdout 64 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61
rdout 65 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61
rdout 66 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66
rdout 67 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66
rdout 68 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68
rdout 69 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69
rdout 70 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69
rdout 71 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69
rdout 72 4 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69
rdout 73 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69 73
rdout 74 10 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69 73
rdout 75 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69 73 75
rdout 76 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69 73 75
rdout 77 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69 73 75
rdout 78 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69 73 75
rdout 79 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69 73 75
rdout 80 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69 73 75
rdout 81 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69 73 75
rdout 82 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 43 44 47 50 51 54 55 57 59 61 66 68 69 73 75 82
rdout 83 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 44 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83
rdout 84 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 44 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84
rdout 85 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 44 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84
rdout 86 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 44 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86
rdout 87 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 44 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87
rdout 88 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 44 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88
rdout 89 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 44 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88
rdout 90 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 44 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88
rdout 91 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 44 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88
rdout 92 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88 92
rdout 93 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88 92
rdout 94 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88 92
rdout 95 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88 92 95
rdout 96 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88 92 95 96
rdout 97 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88 92 95 96 97
rdout 98 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88 92 95 96 97
rdout 99 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88 92 95 96 97
rdout 100 14 17 18 19 20 21 28 32 33 34 35 37 39 40 41 47 50 51 54 55 57 59 61 66 68 69 73 75 82 83 84 86 87 88 92 95 96 97
